### PR TITLE
Job added

### PR DESCRIPTION
**Pull request by PaterSantyago**

### CODE REVIEW

File: src/Lykke.Quintessence.Job.Domain/Lykke.Quintessence.Job.Domain.csproj [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <RootNamespace>Lykke.Quintessence.Domain</RootNamespace>
+        <Version>1.0.1</Version>
+    </PropertyGroup>
+
```
Comment (OttoVT) [inline src/Lykke.Quintessence.Job.Domain/Lykke.Quintessence.Job.Domain.csproj:8]: dotnet pack instructions pls

File: src/Lykke.Quintessence.Job.DomainServices/Lykke.Quintessence.Job.DomainServices.csproj [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <RootNamespace>Lykke.Quintessence.Domain.Services</RootNamespace>
+        <Version>1.0.1</Version>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\Lykke.Quintessence.DomainServices\Lykke.Quintessence.DomainServices.csproj" />
+      <ProjectReference Include="..\Lykke.Quintessence.Job.Domain\Lykke.Quintessence.Job.Domain.csproj" />
+    </ItemGroup>
+
```
Comment (OttoVT) [inline src/Lykke.Quintessence.Job.DomainServices/Lykke.Quintessence.Job.DomainServices.csproj:13]: dotnet pack instructions

File: src/Lykke.Quintessence.Job/Lykke.Quintessence.Job.csproj [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\Lykke.Quintessence.AzureRepositories\Lykke.Quintessence.AzureRepositories.csproj" />
+      <ProjectReference Include="..\Lykke.Quintessence.Job.DomainServices\Lykke.Quintessence.Job.DomainServices.csproj" />
+      <ProjectReference Include="..\Lykke.Quintessence.Job.Domain\Lykke.Quintessence.Job.Domain.csproj" />
+      <ProjectReference Include="..\Lykke.Quintessence.RpcClient\Lykke.Quintessence.RpcClient.csproj" />
+      <ProjectReference Include="..\Lykke.Quintessence.Runtime\Lykke.Quintessence.Runtime.csproj" />
+    </ItemGroup>
+
```
Comment (OttoVT) [inline src/Lykke.Quintessence.Job/Lykke.Quintessence.Job.csproj:16]: dotnet pack instructions

**Review by OttoVT — CHANGES_REQUESTED** [review]

File: src/Lykke.Quintessence.Job.Domain/Lykke.Quintessence.Job.Domain.csproj (hunk 1 above)
Comment (PaterSantyago) [reply src/Lykke.Quintessence.Job.Domain/Lykke.Quintessence.Job.Domain.csproj:8]: fixed

File: src/Lykke.Quintessence.Job/Lykke.Quintessence.Job.csproj (hunk 3 above)
Comment (PaterSantyago) [reply src/Lykke.Quintessence.Job/Lykke.Quintessence.Job.csproj:16]: fixed

File: src/Lykke.Quintessence.Job.DomainServices/Lykke.Quintessence.Job.DomainServices.csproj (hunk 2 above)
Comment (PaterSantyago) [reply src/Lykke.Quintessence.Job.DomainServices/Lykke.Quintessence.Job.DomainServices.csproj:13]: fixed

File: src/Lykke.Quintessence.Job/PeriodicalHandlers/DefaultBalanceCheckScheduler.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        private readonly TimerTrigger _timerTrigger;
+        
+        public DefaultBalanceCheckScheduler(
+            IBalanceCheckSchedulingService balanceCheckSchedulingService,
+            ILogFactory logFactory)
+        {
+            _timerTrigger = new TimerTrigger
+            (
+                nameof(DefaultBalanceCheckScheduler),
+                TimeSpan.FromMinutes(15),
```
Comment (cassade) [inline src/Lykke.Quintessence.Job/PeriodicalHandlers/DefaultBalanceCheckScheduler.cs:23]: Maybe it's better to get value from settings.

**Review by cassade — APPROVED** [review]

File: src/Lykke.Quintessence.Job/PeriodicalHandlers/DefaultBalanceCheckScheduler.cs (hunk 4 above)
Comment (PaterSantyago) [reply src/Lykke.Quintessence.Job/PeriodicalHandlers/DefaultBalanceCheckScheduler.cs:23]: Opened an issue

**Review by OttoVT — APPROVED** [review]

**Pull request merged**